GLOSSARY.md

```diff
@@ -0,0 +1,21 @@
+# Glossary
+
+- **Desired state**
+
+    The aggregate of all configuration data for a system form its "Desired State". The "Desired State" of a system is defined as data sufficient to recreate the system from nothing so that instances of the system are behaviorally indinstinguishable.
```

Comment:
I find ".. from nothing.." too vague, although I don't have a perfect better wording. "from scratch" is probably not much better. Maybe leave this out, and write:

```suggestion
    The aggregate of all configuration data for a system form its "Desired State".  The "Desired State" of a system is defined as data sufficient to recreate the system so that instances of the system are behaviourally indistinguishable.
```